apps/linode-marketplace-wazuh/roles/common/tasks/ufw_rules.yml

```diff
@@ -0,0 +1,61 @@
+---
+# set app specific ufw rules 
+- name: Allow all access to tcp port 22
+  community.general.ufw:
+    rule: allow
+    port: '22'
+    proto: tcp
+
+- name: Allow all access to tcp port 80
+  community.general.ufw:
+    rule: allow
+    port: '80'
+    proto: tcp
+
+- name: Allow all access to tcp port 443
+  community.general.ufw:
+    rule: allow
+    port: '443'
+    proto: tcp
+
+- name: Allow all access to tcp port 1514
+  community.general.ufw:
+    rule: allow
+    port: '1514'
+    proto: tcp
+
+- name: Allow all access to tcp port 1515
+  community.general.ufw:
+    rule: allow
+    port: '1515'
+    proto: tcp
+
+- name: Allow all access to tcp port 1516
+  community.general.ufw:
+    rule: allow
+    port: '1516'
+    proto: tcp
+
+- name: Allow all access to tcp port 514
+  community.general.ufw:
+    rule: allow
+    port: '514'
+    proto: tcp
+
+- name: Allow all access to tcp port 55000
+  community.general.ufw:
+    rule: allow
+    port: '55000'
+    proto: tcp
+
+- name: Allow all access to tcp port 9200
+  community.general.ufw:
+    rule: allow
+    port: '9200'
+    proto: tcp
+
+- name: Allow all access to tcp port 9300
+  community.general.ufw:
+    rule: allow
+    port: '9300'
+    proto: tcp
```

Comment:
```suggestion
```
These ports are listening on localhost. We'll let the customer configure agent for the cluster. I removed these ports and the app works normally as well.